FlowCrypt/Controllers/Threads/ThreadDetailsDecorator.swift

```diff
@@ -11,6 +11,12 @@ import UIKit
 
 extension ThreadMessageSenderCellNode.Input {
     init(threadMessage: ThreadDetailsViewController.Input) {
+        let signature: String
+        if let processedMessage = threadMessage.processedMessage, processedMessage.messageType == .encrypted {
+            signature = processedMessage.signature.message
+        } else {
+            signature = "message_not_signed".localized
+        }
```

Comment:
This is likely the reason why signed-only messages are not recognized as signed - per above code, if they are not encrypted, they are always rendered as "not signed".

Would it be possible to always use `processedMessage.signature.message` regardless if encrypted or not? If that is missing, sounds like an error as well (since when signature is missing, `processedMessage.signature.message` is supposed to be there with `notSigned` case or similar)

Can address later.

Reply:
#1020

Reply:
Yes, initially I thought that all signed messages are encrypted, and haven't implemented signature state check for this case.

Reply:
Check the browser extension - next to the send button, there is a chevron. That's how a signed-only message could be sent.

Reply:
Oh, didn't know about such feature 👍